bl-languages/fi_FI.json

```diff
@@ -0,0 +1,242 @@
+	"emails-will-be-sent-from-this-address":"Sähköpostit tullaan lähettämään tästä osoitteesta.",
+	"bludit-login-access-code": "BLUDIT - Kirjautumisen tunnistautumiskoodi",
+	"check-your-inbox-for-your-login-access-code":"Tunnistautumiskoodi on lähetetty sinulle sähköpostilla",
+	"there-was-a-problem-sending-the-email":"Sähköpostin lähettämisessä tapahtui ongelma",
+	"back-to-login-form": "Takaisin",
+	"send-me-a-login-access-code": "Lähetä minulle tunnistautumiskoodi",
+	"get-login-access-code": "Lähetä tunnistautumiskoodi",
+	"email-notification-login-access-code": "<p>Tämä on ilmoitus sivustoltasi {{WEBSITE_NAME}}</p><p>Jos pyysit kirjautumisen tunnistautumiskoodia, paina tästä linkistä:</p><p>{{LINK}}</p>",
+	"there-are-no-scheduled-posts": "Ei ajastettuja blogikirjoituksia",
+	"show-password": "Näytä salasana",
+	"edit-or-remove-your=pages": "Lisää, muokkaa ja poista sivujasi",
+	"edit-or-remove-your-blogs-posts": "Lisää, muokkaa ja poista blogikirjoituksiasi",
+	"general-settings": "Yleiset asetukset",
+	"advanced-settings": "Lisäasetukset",
+	"manage-users": "Hallitse käyttäjiä",
+	"view-and-edit-your-profile": "Näytä ja muokkaa oma profiilisi",
+
+	"password-must-be-at-least-6-characters-long": "Salasanan täytyy olla vähintään 6 merkkiä pitkä",
+	"images": "Kuvat",
+	"upload-image": "Lähetä image",
+	"drag-and-drop-or-click-here": "Klikkaa tästä, tai vedä kuva tänne",
+	"insert-image": "Lisää kuva",
+	"supported-image-file-types": "Tuetut kuvatyypit",
+	"date-format": "Päivämäärän muoto",
+	"time-format": "Ajan muoto",
+	"chat-with-developers-and-users-on-gitter":"Keskustele muiden kehittäjien ja käyttäjien kanssa [Gitterissä](https://gitter.im/dignajar/bludit)",
+	"this-is-a-brief-description-of-yourself-our-your-site":"Tämä on lyhyt kuvaus sinusta, tai sivustostasi. Voit muuttaa tätä hallintapaneelista kohdasta Asetukset->Lisäosat, ja muokkaa lisäosan \"About\" asetuksia",
+	"profile-picture": "Profiilikuva",
+	"the-about-page-is-very-important": "Tietoja-sivu on tärkeä työkalu mahdollisille asiakkaillesi. Niille asiakkaille, jotka miettivät, kuka on tämän mahtavan sivuston takana, sinun Tietoja-sivu on ensimmäinen paikka josta katsoa.",
+	"change-this-pages-content-on-the-admin-panel": "Voit muokata tämän sivun sisältöä hallintapaneelista, kohdasta Hallitse->Sivut, ja paina \"Tietoja\".",
+	"about-your-site-or-yourself": "Sivustostasi tai sinusta.",
+	"welcome-to-bludit": "Bludit toivottaa sinut tervetulleeksi!",
+
+	"site-information": "Sivuston tiedot",
+	"date-and-time-formats": "Päivämäärän ja ajan esitysmuodot",
+	"activate": "Ota käyttöön",
+	"deactivate": "Poista käytöstä",
+
+	"cover-image": "Artikkelikuva",
+	"blog": "Blogi",
+	"more-images": "Lisää kuvia",
+
+	"click-here-to-cancel": "Paina tästä peruaksesi.",
+	"type-the-tag-and-press-enter": "Kirjoita tagi, ja paina enter.",
+	"add": "Lisää",
+	"manage-your-bludit-from-the-admin-panel": "Voit hallita Bludit:ia [hallintapaneelista]({{ADMIN_AREA_LINK}})",
```

Comment:
"manage-your-bludit-from-the-admin-panel": "Voit hallita Bluditia [hallintapaneelista]({{ADMIN_AREA_LINK}})",